recipes/csvtk/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
+  name: csvtk
+  version: 0.4.4
+
+about:
+  home: https://github.com/shenwei356/csvtk
+  summary: another cross-platform, efficient, practical and pretty CSV/TSV toolkit
+  license: MIT
+
+build:
+  number: 0
+  skip: False
+
+source:
+  fn: csvtk_v0.4.4_osx.tar.gz # [osx]
+  url: https://github.com/shenwei356/csvtk/releases/download/v0.4.4/csvtk_darwin_amd64.tar.gz # [osx]
```

Comment:
csvtk, like seqkit which is already in this recipe, is written in Go language so it can be cross-compiled to executable binaries for major OSs. The static-linked binaries have no dependencies so people can immediatly run it after download and decompression. Therefore, there's no need to compile again.